examples/mysql/1.7/mysql.cnf

```diff
@@ -43,4 +43,7 @@ log_bin           = mysql-bin
 expire_logs_days  = 1
 binlog_format     = row
 
+# Configures the amount of table metadata added to the binary log when using row-based logging
```

Comment:
I don't think we should change this here; instead, we could provide a specific mysql.conf file just for the single integration test (using Testcontainers) related to invisible columns.